### PR TITLE
docs: update PR title guideline and templates (DSP-1189)

### PR DESCRIPTION
**Pull request by mpro7**
resolves DSP-1189

### CODE REVIEW

File: docs/developers/dsp/contribution/index.md [hunk 1]
```diff
@@ -72,16 +72,16 @@ Since we started to use the [release-please-action](https://github.com/marketpla
 #### PR Title Format
 
 ```text
-<type>(<DSP-no.>): <subject>
+<type>(<scope>): <subject> (<DSP-no.>)
 ```
 
-It's crucial to start the PR title with the `<type>` ([allowed types](http://localhost:8000/developers/dsp/contribution/#type)), followed by `<DSP-no.>`, which represents the number of the task(s) related to the PR. `<subject>` should be YouTrack task title or its short version started with the small letter. Here is the example:
+It's crucial to start the PR title with the `<type>` ([allowed types](http://localhost:8000/developers/dsp/contribution/#type)), followed by optional`<scope>`. `<subject>` should be YouTrack task title or its short version. At the end of the title PR title add inside the brackets `<DSP-no.>`, which represents the number of the task(s) related to the PR. Here is the example:
```
Comment (kilchenmann) [inline docs/developers/dsp/contribution/index.md:78]: Didn't saw it in the last PR, but the URL in allowed types seems to be wrong 😉

Comment (kilchenmann) [reply docs/developers/dsp/contribution/index.md:78]: You can add internal link just like:

`[allowed types](#type)`

Comment (kilchenmann) [reply docs/developers/dsp/contribution/index.md:78]: There's another wrong localhost link, which I couldn't find here anymore...
`above` in this section: https://docs.dasch.swiss/developers/dsp/contribution/#pull-request-guidelines

**Review by kilchenmann — CHANGES_REQUESTED** [review]

File: docs/developers/dsp/contribution/index.md (hunk 1 above)
Comment (mpro7) [reply docs/developers/dsp/contribution/index.md:78]: Good catch, thanks ;).

**Review by kilchenmann — APPROVED** [review]

**Pull request merged**